selenium_tests/course_info/course_info_is_initialized_test.py

```diff
@@ -0,0 +1,34 @@
+import unittest
+from selenium_tests.course_info.course_info_base_test_case import CourseInfoBaseTestCase
+from selenium_tests.course_info.page_objects.course_info_parent_page_object import CourseInfoParentPage
+
+
+class CourseInfoIsInitializedTest(CourseInfoBaseTestCase):
+
+    def test_is_page_loaded(self):
+
+        """
+        Check if page is loaded
+        """
+        driver = self.driver
+        # initialize
+        page = CourseInfoParentPage(driver)
+        page.is_loaded()
+
+    def test_is_link_present(self):
+        """
+        Check that Course Info link is loaded by checking against Link text
+        """
+        driver = self.driver
+        page = CourseInfoParentPage(driver)
+
+        # verify that the course info block is present
+        course_info_link = page.get_course_info_link()
```

Comment:
I think it would be better to encapsulate this in the page object, e.g. have the page object present a method which checks this stuff (page element details, etc), and here all that's needed is `self.assertTrue(is_course_info_block_present())`.